markdownlint.js

```diff
@@ -161,7 +161,37 @@ function printResult(lintResult) {
     process.exitCode = 1;
   }
 
-  if (program.output) {
+  if (program.junit) {
+    const builder = require('junit-report-builder');
+    const suite = builder
+      .testSuite()
+      .name('Markdownlint')
+      .timestamp(new Date().toISOString());
+    if (results.length > 0) {
+      results.forEach(result => {
+        const {file, lineNumber, column, names, description} = result;
+        const columnText = column ? `:${column}` : '';
+        suite
+          .testCase()
+          .className(file)
+          .name(`${file}:${lineNumber}${columnText} ${names}`)
+          .failure(`${file}:${lineNumber}${columnText} ${names} ${description}`, names)
+          .time(0);
+      });
+    } else {
+      suite.testCase().className(program.args).name('Markdownlint');
```

Comment:
Why program.args here?

Reply:
My understanding is that program.args will be "everything else" in the command line that isn't part of an option, so in the case of markdownlint-cli it should be what's specified to be linted.  The className argument seems to generally be the "file" that the test was run against, and in the absence of errors I thought the files being linted would make sense as the "class name" but I'm open other suggestions here as well.

Reply:
Makes sense!